dot/state/base.go

```diff
@@ -155,26 +154,3 @@ func (s *BaseState) loadSlotDuration() (uint64, error) {
 
 	return binary.LittleEndian.Uint64(data), nil
 }
-
-// storePruningData stores the pruner configuration.
-func (s *BaseState) storePruningData(mode pruner.Config) error {
```

Comment:
That was persisting the pruning mode, whereas really it should be given at start from the toml config or flags, and be able to change from one mode to the other.